frontend/src/index.css

```
@@ -111,51 +133,45 @@ body {
+  text-indent: 0 !important;
 }
 
-.table-wrapper td {
-  white-space: nowrap;
+.legacy-html-table ul,
+.legacy-html-table ol {
+  margin: 0 !important;
+  padding-left: 1.25rem;
 }
 
 @media (max-width: 480px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)